client/me/purchases/confirm-cancel-purchase/index.jsx

```diff
@@ -66,6 +67,12 @@ const ConfirmCancelPurchase = React.createClass( {
 		}
 
 		notices.success( response.message, { persistent: true } );
+
+		analytics.tracks.recordEvent(
+			'calypso_purchases_submit_cancel_form',
```

Comment:
Super minor, but I believe this should be `calypso_purchases_cancel_form_submit` per the analytics readme:

> In order to keep similar events grouped together when output in an alphabetized list (as is typical with ananlytics tools), put the verb at _the end_ of the event name